.pipelines/pr-windows-signed-scripts.yaml

```diff
@@ -0,0 +1,65 @@
+trigger: none
+pr: none

```

Comment:
should this be on?

Reply:
No, we override these when we make the pipelines in Azure DevOps.